### DISCUSSION TITLE
阿里云盘token管理问题

### DISCUSSION THREAD

**Discussion by XiaDao** [AlistGo/alist#2630] [Q&A]
用同一个阿里云盘账号token，每添加一个阿里云盘分享链接，都要获取一个新的token，目前每一个token都是单独管理的么？假如未来增加了100+个分享链接挂载，是不是意味着要维护100个token的保活，目前有没有统一管理同一个账号共用同一个token的机制？token过多会不会触发某些风控机制

**Comment by nxjonson** [comment]
一个token当天总是有效。为了保密，每次刷新alist内会显示不同。别当真

**Reply by XiaDao** [reply]
感谢，解答了我的困惑！👍
